Reconstruction/reconstruction.tex

```diff
@@ -330,7 +331,7 @@ \subsection{Isolated Photons}
   \end{center}
 \end{table}
 
-\subsection{Hadrons}
+\subsection{Hadrons} \PH{I would move this earlier, since the object ids occur after PF. Also you might want to label this PF or something closer to that. }
```

Comment:
I'm cheating a little and merging PF with object IDs. also defined isolations before I defined hadrons... but muons, electrons, and photons are formed before pf hadrons.

I basically wanted to avoid going

muons, electrons, photons, hadrons, isolation, muon IDs, electron IDs, photon IDs, jets, met because it feels redundant.

I think I clarify that the further IDs are for the analysis and not for the PF algorithm